include/deal.II/base/synchronous_iterator.h

```diff
@@ -38,6 +41,12 @@ DEAL_II_NAMESPACE_OPEN
  * The template argument of the current class shall be of type
  * <code>std_cxx11::tuple</code> with arguments equal to the iterator types.
  *
+ * The individual iterators can be accessed using 
+ * <code>std_cxx1::get<X>(synchronous_iterator.iterators)</code> where X is the 
+ * number corresponding to the desired iterator.
+ *
+ * Additional documentation can be found under Files then File Members.
```

Comment:
The problem is that right now, I can't find the documentation for the global functions anywhere. By adding the ```@files``` the documentation is generated but it is hard to find.